apps/charterafrica/src/payload/blocks/Consultation.js

```diff
@@ -0,0 +1,119 @@
+import {
+  mapPlaylistLinkToId,
+  mapVideoUrlToId,
+} from "../utils/mapPlaylistLinkToId";
+
+const Consultations = {
+  slug: "consultation",
+  labels: {
+    singular: {
+      en: "Consultation Playlist",
+      fr: "Playlist de consultation",
+      pt: "Consulta Lista de reprodução",
+    },
+    plural: {
+      en: "Consultation Playlists",
+      fr: "Playlist de consultations",
+      pt: "Lista de reprodução de consulta",
+    },
+  },
+  fields: [
+    {
+      type: "collapsible",
+      label: {
+        en: "Playlist",
+        fr: "Playlist",
+        pt: "Lista de reprodução",
+      },
+      fields: [
+        {
+          name: "title",
+          label: {
+            en: "Title",
+            fr: "Titre",
+            pt: "Título",
+          },
+          type: "text",
+          required: true,
+          localized: true,
+        },
+        {
+          name: "playlistLink",
+          label: {
+            en: "Playlist",
+            fr: "Playlist",
+            pt: "Lista de reprodução",
+          },
+          type: "text",
+          required: true,
+          localized: true,
+        },
+        {
+          name: "playlistId",
+          label: {
+            en: "Playlist ID",
+            fr: "Identifiant de la liste de lecture",
+            pt: "ID da lista de reprodução",
+          },
+          admin: {
+            hidden: true,
+          },
+          hooks: {
+            beforeValidate: [mapPlaylistLinkToId],
+          },
+          type: "text",
+          localized: true,
+        },
+      ],
+    },
+    {
+      name: "featuredConsultations",
```

Comment:
```suggestion
      name: "featured",
```
1. Since this is already Consultation block, featured is enough. No need to repeat consultation.
2. I'm just thinking out loud but I think may be we should offer option of featured being latest/most recent or custom. When custom is selected, that's when we ask for video url